parameter/ConfigurableDomain.h

```diff
@@ -148,12 +188,26 @@ class CConfigurableDomain : public CBinarySerializableElement
     // Check configurable element already attached
     bool containsConfigurableElement(const CConfigurableElement* pConfigurableCandidateElement) const;
 
-    // Merge any descended configurable element to this one
-    void mergeAlreadyAssociatedDescendantConfigurableElements(CConfigurableElement* pNewConfigurableElement);
+    /** Merge any descended configurable element to this one
+     *
+     * @param[in] newElement pointer to element which has potential descendants which can be merged
+     * @param[out] infos useful information we can provide to client
+     */
+    void mergeAlreadyAssociatedDescendantConfigurableElements(CConfigurableElement* newElement,
+                                                              core::Results& infos);
+
     void mergeConfigurations(CConfigurableElement* pToConfigurableElement, CConfigurableElement* pFromConfigurableElement);
 
-    // Configurable elements association
-    void doAddConfigurableElement(CConfigurableElement* pConfigurableElement, const CParameterBlackboard* pMainBlackboard = NULL);
+    /** Actually realize the association between the domain and a configurable  element
+     *
+     * @param[in] pConfigurableElement pointer to the element to add
+     * @param[out] infos useful information we can provide to client
+     * @param[in] pMainBlackboard pointer to the application main blackboard
```

Comment:
Please explain what is supposed to happen if pMainBlackboard == NULL. If there is a way to tell doxygen that the argument is optional, please use it (but maybe doxygen already knows this by parsing the prototype).